midend/removeParameters.cpp

```diff
@@ -76,6 +76,12 @@ void FindActionParameters::postorder(const IR::MethodCallExpression* expression)
     }
 }
 
+const IR::Node* DoRemoveActionParameters::preorder(IR::EntriesList *list)
```

Comment:
You MUST remove action parameters - this takes care of the data-plane bound parameters.
This function should be removed; it may work correctly if you make the change above.